wdl/GermlineCNVTasks.wdl

```diff
@@ -18,10 +18,12 @@ task AnnotateIntervals {
       RuntimeAttr? runtime_attr_override
     }
 
+    Float input_size = size(select_all([intervals, ref_fasta, mappability_track_bed, segmental_duplication_track_bed]), "GB")
+
     RuntimeAttr default_attr = object {
       cpu_cores: 1,
       mem_gb: 1.7,
-      disk_gb: 10,
+      disk_gb: ceil(10.0 + input_size),
```

Comment:
Is 10GB enough for the docker image and all outputs?  Might also want to multiply it by 1.15, since Azure Batch requires 10% of the disk as well: `disk_gb: ceil((10.0 + input_size) * 1.15)`

Reply:
That is a good point!

Reply:
On GCP, the docker image goes on a separate boot disk

Reply:
I merged this before having pushed a commit with @MattMcL4475's suggestion. Hopefully, it will work with the increased disk size; otherwise, I will make a follow-up PR.